scripts/jobs/cron_tasks.inc.http.10.apache.php

```diff
@@ -557,18 +557,35 @@ protected function composePhpOptions($domain, $ssl_vhost = false)
 			$php_options_text .= '  php_admin_value sendmail_path "/usr/sbin/sendmail -t -f ' . $cmail . '"' . PHP_EOL;
 			
 			if ($domain['openbasedir'] == '1') {
-				if ($domain['openbasedir_path'] == '1' || strstr($domain['documentroot'], ":") !== false) {
-					$_phpappendopenbasedir = appendOpenBasedirPath($domain['customerroot'], true);
+				/***
+					0   - documentroot
+					1   - customerroot
+					2   - parent of documentroot
+					*:* - customerroot
+				***/
+				if ($this->_domain['openbasedir_path'] == '2'
```

Comment:
there is no class-member called "_domains"...did you just copy/paste from fcgid/fpm?